### PR TITLE
bug fix for hltIntegration tests when using a setup menu : 12_3_X

### PR DESCRIPTION
**Pull request by Sam-Harper**
#### PR description:

There was a bug in the hltIntegration tests where the setup menu was overridden with the main menu when trying to determine the database and converter parameters. Thanks to AR Sahasransu for reporting. 

#### PR validation:

the following command now completes successfully

```
hltIntegrationTests /users/asahasra/Run3HLTv1LLP/HLT_Mu20NoFiltersNoVtxDisplaced_Photon20CaloCustomTightId --setup /dev/CMSSW_12_3_0/GRun/V15 -i file:/eos/cms/store/data/Run2018D/EphemeralHLTPhysics1/RAW/v1/000/323/775/00000/0244D183-F28D-2741-9DBF-1638BEDC734E.root -x "--globaltag auto:run3_hlt --customise HLTrigger/Configuration/customizeHLTforCMSSW.customiseFor2018Input"
```

### CODE REVIEW

File: HLTrigger/Configuration/scripts/hltIntegrationTests [hunk 1]
```diff
@@ -332,7 +332,7 @@ if [ "$SETUP" ]; then
   #we use $MENU not $SETUP here as we force the same DB / converter as the main menu
   #this is the hltGetConfiguration behaviour and would be confusing if you had to 
   #specify converter/db on the setup menu on hltIntegration tests but not on hltGetConfiguration
-  read SETUP_Vx SETUP_DB SETUP <<< $(parse_HLT_menu "$MENU")
+  read SETUP_Vx SETUP_DB CFG <<< $(parse_HLT_menu "$MENU")
```
Comment (missirol) [inline HLTrigger/Configuration/scripts/hltIntegrationTests:335]:
```suggestion
  read SETUP_Vx SETUP_DB _ <<< $(parse_HLT_menu "$MENU")
```
Optional change: iiuc, `CFG` does not exist before, and would not be used elsewhere after this line; with `_` we could obtain the same, avoiding the edge case where `CFG` gets overwritten (if one day a `CFG` variable makes it into these scripts).

**Comment by missirol** [comment]
type bugfix

File: HLTrigger/Configuration/scripts/hltIntegrationTests (hunk 1 above)
Comment (Sam-Harper) [reply HLTrigger/Configuration/scripts/hltIntegrationTests]: sounds good

**Comment by missirol** [comment]
please test

Tested locally, and works as expected. Thanks for the fix, Sam.

**Comment by missirol** [comment]
+hlt

**Comment by qliphy** [comment]
+1

**Pull request merged**